web/e2e/tests/projects.spec.ts

```
@@ -0,0 +1,121 @@
+      }
+    });
+    page = await context.newPage();
+    loginPage = new LoginPage(page);
+    dashBoardPage = new DashBoardPage(page);
+    projectsPage = new ProjectsPage(page);
+    projectScreen = new ProjectScreenPage(page);
+
+    await page.goto(REEARTH_WEB_E2E_BASEURL, { waitUntil: "networkidle" });
+  });
+
+  test.afterEach(async ({}, testInfo) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)